FireboltNETSDK/Client/FireboltConnection.cs

```diff
@@ -316,6 +384,7 @@ public override void Close()
         {
             _connectionState.State = ConnectionState.Closed;
             Client = null;
+            _isSystem = true;
```

Comment:
Why do we set connection to system when we close it?

Reply:
We have to restore the state. Otherwise we cannot open connection again. Take a look on line `if (_accountId == null && Account != null && _isSystem)`.
I spent a lot of time debugging this.